src/map/script.cpp

```diff
@@ -17385,6 +17385,17 @@ BUILDIN_FUNC(checkidle)
 	return SCRIPT_CMD_SUCCESS;
 }
 
+BUILDIN_FUNC(checkidlehom)
+{
+	TBL_PC *sd = NULL;
+
+	if( script_nick2sd(2,sd) )
+		script_pushint(st, DIFF_TICK(last_tick, sd->idletime_hom));
```

Comment:
Shouldnt this simply return pc_isidle_hom?

Reply:
No, that would only return if the player is idle or not, not the idle time